VitDeck/Assets/VitDeck/Validator/Tests/TestValidationTargetFinder.cs

```diff
@@ -77,13 +76,48 @@ public void TestFindAssetObjects()
             //Sample_object.fbx 
             Assert.That(Array.Exists(assetObjects, obj => AssetDatabase.GetAssetPath(obj) == (testFolder + "/Sample_object.fbx")), Is.True);
             //Sub Fonder
-            Assert.That(Array.Exists(assetObjects, obj => AssetDatabase.GetAssetPath(obj) == (testFolder + "/Sub Fonder")), Is.True);
-            //Scene in sub folder.unity
-            Assert.That(Array.Exists(assetObjects, obj => AssetDatabase.GetAssetPath(obj) == (testFolder + "/Sub Fonder/Scene in sub folder.unity")), Is.True);
+            Assert.That(Array.Exists(assetObjects, obj => AssetDatabase.GetAssetPath(obj) == (testFolder + "/Sub Folder")), Is.True);
+            //New Prefab in sub folder.prefab
+            Assert.That(Array.Exists(assetObjects, obj => AssetDatabase.GetAssetPath(obj) == (testFolder + "/Sub Folder/New Prefab in sub folder.prefab")), Is.True);
             //FBXのサブアセットによりオブジェクトが重複取得されない
             Assert.That(assetObjects.Length, Is.EqualTo(assetObjects.Distinct().Count()));
         }
         [Test]
+        public void TestFindScenes()
+        {
+            var testFolder = "Assets/VitDeck/Validator/Tests/ValidationTargetFinder";
+            var finder = new ValidationTargetFinder();
+            var scenes = finder.FindScenes(testFolder, true);
+            //New Scene
+            Assert.That(scenes.Length, Is.EqualTo(1));
+            Assert.That(Array.Exists(scenes, scene => scene.path == "Assets/VitDeck/Validator/Tests/ValidationTargetFinder/New Scene.unity"), Is.True);
+        }
+        [Test]
+        public void TestFindRootObjects()
+        {
+            var testFolder = "Assets/VitDeck/Validator/Tests/ValidationTargetFinder";
+            var finder = new ValidationTargetFinder();
+            var rootObjects = finder.FindRootObjects(testFolder, true);
+            //New Scene
+            Assert.That(rootObjects.Length, Is.EqualTo(3));
+            Assert.That(Array.Exists(rootObjects, obj => obj.name == "Cube(main)"), Is.True);
+            Assert.That(Array.Exists(rootObjects, obj => obj.name == "Directional Light"), Is.True);
+            Assert.That(Array.Exists(rootObjects, obj => obj.name == "Main Camera"), Is.True);
+        }
+        [Test]
+        public void TestFindObjects()
+        {
+            var testFolder = "Assets/VitDeck/Validator/Tests/ValidationTargetFinder";
+            var finder = new ValidationTargetFinder();
+            var allObjects = finder.FindAllObjects(testFolder);
```

Comment:
```suggestion
            var allObjects = finder.FindAllObjects(testFolder, true);
```
ここだけテスト中にダイアログが出るようになっています。

Reply:
見落としてました、ありがとうございます。
こちらコミット後にテスト通ることを確認してマージします！